pipeline-maven/src/test/java/org/jenkinsci/plugins/pipeline/maven/DependencyFingerprintPublisherTest.java

```diff
@@ -28,18 +26,18 @@ public void verify_fingerprinting_of_dependencies() throws Exception {
 
         loadMonoDependencyMavenProjectInGitRepo(this.gitRepoRule);
 
-        //@formatter:off
-        String pipelineScript = "node() {\n" +
-            "    git($/" + gitRepoRule.toString() + "/$)\n" +
-            "    withMaven(options:[dependenciesFingerprintPublisher(includeReleaseVersions:true)]) {\n" +
-            "        if (isUnix()) {\n" +
-            "            sh 'mvn package'\n" +
-            "        } else {\n" +
-            "            bat 'mvn package'\n" +
-            "        }\n" +
-            "    }\n" +
-            "}";
-        //@formatter:on
+        // @formatter:off
+        String pipelineScript = "node() {\n" + "    git($/"
+                + gitRepoRule.toString() + "/$)\n"
+                + "    withMaven(options:[dependenciesFingerprintPublisher(includeReleaseVersions:true)]) {\n"
+                + "        if (isUnix()) {\n"
+                + "            sh 'mvn package'\n"
+                + "        } else {\n"
+                + "            bat 'mvn package'\n"
+                + "        }\n"
+                + "    }\n"
+                + "}";
+        // @formatter:on
```

Comment:
Why those changes ? spotless does not understand the `formatter:off` tag ?

(and I know, this is ugly ... need java 17 and text block)

Reply:
based on your comment from https://github.com/jenkinsci/pipeline-maven-plugin/pull/715#discussion_r1343512520
then I changed all to use the same style everywhere.
yes spotless does not understand.
is it really a problem?

Reply:
well ... as I said, now it is ugly, with automatic spotless format, I find it worse:
```
String pipelineScript = "node() {\n" + "    git($/"
                + gitRepoRule.toString() + "/$)\n"
```
instead of
```
        String pipelineScript = "node() {\n" +
            "    git($/" + gitRepoRule.toString() + "/$)\n" +
```

I also think that while an automated format is good, not having an option to manually bypass for some pieces of code will miss.

Seem just a little configuration to add:
```
<toggleOffOn>
  <off>@formatter:off</off>
  <on>@formatter:on</on>
</toggleOffOn>
```
https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md#spotlessoff-and-spotlesson

Reply:
TBH it's just a matter of `+` at the start or the end. no much real difference :)
but if you consider this really ugly you will have to write a more complicated pom.xml. 
your choice. 
at the end of day you just run `spotless:check` and/or `spotless:apply` and voila

Reply:
That's not just a matter of  `+` at the start or end

Without spotless changes, one line of code equals one line of the executed pipeline script. Not it is not the case, look at the `git`  step call

Reply:
ah right. ok we will turn into an ugly pom :)

Reply:
if you have a better solution, I am in !

Reply:
haha nope :)